packages/deployment/app/cumulus_api.template.yml

```diff
@@ -178,6 +178,9 @@ Resources:
       Environment:
         Variables:
           stackName: {{../stackName}}
+          system_bucket: {{../parent.system_bucket}}
+          public_buckets: {{#collectBuckets ../parent.buckets "public"}}{{/collectBuckets}}
```

Comment:
You don't need {{/collectBuckets}}, you should definitely use the "{{{" invocation.   See "ToJson" .     What was the failure when you used this invocation?     What was the resultant template/failure?

Reply:
Er, and yes, the handlebars helpers are only applied to the template object. 

See (in parseCF, in kes.js) 

```   const template = Handlebars.compile(t);
    return template(this.config);```

The config object interpolation of things like {{EARTHDATA_CLIENT_ID}} is being handled in kes's config.js (parseConfig).     As an aside, config.js is *not* overridable due to how the cli/index brings in the config before loading the kes class.